### PR TITLE
Add link to My Page from header

### LINKED ISSUE #108: Link avatar to My Page
The avatar in the upper right corner after logging in should link to "My Page" (`/my`). Currently it doesn't.

image

 Steps to reproduce:

1. Navigate to www.dev.zetkin.org (after setting up the development environment)
2. Click the login button
3. Log in using username `testadmin@example.com` and password `password`
4. Once back at www.dev.zetkin.org, click the avatar in the top right corner

Expected results: Should navigate to /my
Actual results: Does nothing

### PR DESCRIPTION
**Pull request by ziggabyte**
Added link to My Page inside the View element. Fixes #108

### CODE REVIEW

**Review by richardolsson — CHANGES_REQUESTED** [review]
Great work! This looks completely flawless. Only thing missing now is a test. Let's write one together!

**Comment by ziggabyte** [comment]
Added a test to log in and click on user avatar, to verify that it takes you to the correct url.
NOTE: this commit also contains a change in package.json where tdd now has escaped " instead of plain '

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "devproxy": "moxy -f http://api.dev.zetkin.org",
     "devserver": "next dev -p 80",
     "lint": "eslint src cypress && tsc --noEmit && echo 'Finished linting without errors!'",
-    "tdd": "concurrently 'NODE_ENV=test yarn devserver' 'yarn devproxy' 'yarn cypress open'",
+    "tdd": "concurrently \"NODE_ENV=test yarn devserver\" \"yarn devproxy\" \"yarn cypress open\"",
```
Comment (richardolsson) [inline package.json:10]: Is this actually working now? Based on our discussion on Slack, I don't think this was a fully functional solution, but only solved part of the problem. If it's not working, it should ideally not have been part of the commit. Let's investigate together!

File: cypress/integration/login.spec.ts [hunk 2]
```diff
@@ -18,6 +18,22 @@ describe('Login process', () => {
...
+        cy.visit('/login');
+        cy.get('input[aria-label="E-mail address"]').type('testadmin@example.com');
+        cy.get('input[aria-label="Password"]').type('password');
+        cy.get('input[aria-label="Log in"]')
+            .should('be.visible')
+            .click();
+        cy.url().should('match', /\/$/);
+        cy.get('[data-test="username"').should('be.visible');
+        cy.get('[data-test="user-avatar"]')
+            .should('be.visible')
```
Comment (richardolsson) [inline cypress/integration/login.spec.ts:32]: This line is not necessary. The test before this one already checks that it's visible, and here we don't really care about that, but are instead just interested in making sure that when the link is clicked, it leads to the right URL.

**Review by richardolsson — CHANGES_REQUESTED** [review]
Great! Another great step. :)

I don't think this is working correctly on Windows still. But I guess we could merge it and create a separate pull request to really solve the issues on Windows.

I have one comment below about a redundant line. Please remove that and we should be ready to approve.

**Comment by ziggabyte** [comment]
Removed parts of test that were already performed in other tests, and un-did the change to package.json

File: cypress/integration/login.spec.ts [hunk 3]
```diff
@@ -18,6 +18,16 @@ describe('Login process', () => {
         cy.get('[data-test="username"]').should('be.visible');
         cy.get('[data-test="user-avatar"]').should('be.visible');
     });
+
+    it('clicking on user avatar takes you to My Page', () => {
```
Comment (richardolsson) [inline cypress/integration/login.spec.ts:22]: You should be able to read this line as a sentence, including the `it` in the function name. A suitable description might be "(it) links the user avatar to My Page" or something like that.

This is a minor thing, and not something that needs to be fixed for this to be merged.

**Review by richardolsson — APPROVED** [review]
Great! I feel ready to merge this now. Let's spend some time on getting everything up and running more smoothly on Windows next!

File: cypress/integration/login.spec.ts (hunk 3 above)
Comment (ziggabyte) [reply cypress/integration/login.spec.ts:22]: aaah! got it!

**Pull request merged**